containers/azure-terraform-0.12/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,39 @@
+{
+	"name": "Azure Terraform",
```

Comment:
This will appear in the dropdown in VS Code. Given there's two now, this should probably include the version number correct?

Reply:
Fixed in 30d13b9